src/components/common/CustomHamburgerMenu.js

```diff
@@ -28,13 +29,13 @@ const CustomHamburgerMenu = ({ isLoggedIn }) => {
         <HamburgerMenu
           isOpen={open}
           menuClicked={() => setOpen(!open)}
-          width={hambWidth}
-          height={hambHeight}
-          strokeWidth={hambStrokeWidth}
-          rotate={hambRotate}
+          width={Number(hambWidth)}
+          height={Number(hambHeight)}
+          strokeWidth={Number(hambStrokeWidth)}
+          rotate={Number(hambRotate)}
           color={hambColor}
-          borderRadius={hambBorderRadius}
-          animationDuration={hambAnimationDuration}
+          borderRadius={Number(hambBorderRadius)}
+          animationDuration={Number(hambAnimationDuration)}
```

Comment:
isn't it better to have these as numbers in `_constants`?

Reply:
They are, but when I import this is transformed to string. Is there another way?
what I do is this:

```
// Hamburger menu style
$hambWidth: 20;
$hambHeight: 12;
$hambStrokeWidth: 2;
$hambRotate: 0;
$hambColor: $black;
$hambBorderRadius: 0;
$hambAnimationDuration: 0.5;

:export {
  ...
  hambWidth: $hambWidth;
  hambHeight: $hambHeight;
  hambStrokeWidth: $hambStrokeWidth;
  hambRotate: $hambRotate;
  hambColor: $hambColor;
  hambBorderRadius: $hambBorderRadius;
  hambAnimationDuration: $hambAnimationDuration;
}
```

Reply:
I guess you will use those constants only in js and more specifically in this component, What about defining they as a js constant on the top of the component? You will save from cast to Number and increment the _constant.scss file size unnecessary.